garak/probes/base.py

```diff
@@ -85,18 +85,28 @@ def __init__(self, config_root=_config):
         self.langprovider = self._get_langprovider()
         if self.langprovider is not None and hasattr(self, "triggers"):
             # check for triggers that are not type str|list or just call translate_triggers
+            preparation_bar = tqdm.tqdm(
+                total=len(self.triggers),
+                leave=False,
+                colour=f"#{garak.resources.theme.PROBE_RGB}",
```

Comment:
we should probably get a different colour for this! how are you with green or grey?

Reply:
I used `PROBE_RGB` as these are all presented in probes, green might work, may not be color blind friendly.